internal/authz/oidc_test.go

```diff
@@ -1543,6 +1550,14 @@ func requireStandardResponseHeaders(t *testing.T, resp *envoy.CheckResponse) {
 	}
 }
 
+func oidcConfig(oidc *oidcv1.OIDCConfig) *configv1.Config {
```

Comment:
super nit: this looks more like `config` or `baseConfig` than `oidcConfig` 🤔

Reply:
Done :)